jax/__init__.py

```diff
@@ -126,6 +126,7 @@
 from jax._src.api import vmap as vmap
 from jax._src.api import xla_computation as xla_computation
 from jax._src.sharding_impls import NamedSharding as NamedSharding
+from jax._src.include_dir import include_dir as include_dir
```

Comment:
I'm wondering if we should use a new submodule for this, perhaps a submodule related to defining foreign functions.

Reply:
I think that's a good suggestion! I'll update the PR to have this function in `jax.ffi` perhaps. I don't think that there's currently have anything else that would go in there, but I think that it clarifies the use case.

Another option would be to call the submodule something like `custom_call`, since I expect that we will be adding more infrastructure to support custom calls, and that would be the main interface to the FFI.